extensions/ql-vscode/src/codeql-cli/distribution.ts

```diff
@@ -475,19 +494,41 @@ class ExtensionSpecificDistributionManager {
   }
 
   private createReleasesApiConsumer(): ReleasesApiConsumer {
-    const ownerName = this.config.ownerName
-      ? this.config.ownerName
-      : DEFAULT_DISTRIBUTION_OWNER_NAME;
-    const repositoryName = this.config.repositoryName
-      ? this.config.repositoryName
-      : DEFAULT_DISTRIBUTION_REPOSITORY_NAME;
     return new ReleasesApiConsumer(
-      ownerName,
-      repositoryName,
+      this.distributionOwnerName(),
+      this.distributionRepositoryName(),
       this.config.personalAccessToken,
     );
   }
 
+  private distributionOwnerName(): string {
```

Comment:
Very optional: Since these methods don't have any side effects, it might be a bit nicer to make them getters so you don't need to call into them:

```suggestion
  private get distributionOwnerName(): string {
```

The same could be done for `distributionRepositoryName` and `usingNightlyReleases`.

Reply:
Sounds good to me. I'll change them all to use `get`